### ISSUE TITLE
Request for finetuned AIpparel on GCD-MM

### ISSUE THREAD

**Issue by yangmingweiwtu** [georgeNakayama/AIpparel-Code#19]
Hi, authors,
Thank you for your excellent work on AIpparel. Your ablation experiments compare the model with LoRa fine-tuning. Due to limited lab resources, we cannot fine-tune all parameters. Could you please provide the LoRa fine-tuning code? Finally, why is the performance difference between the LoRa fine-tuning and full parameter fine-tuning models so significant,especially regarding stitch accuracy??

**Comment by georgeNakayama** [comment]
Hi, the LoRA-finetuning experiment was conducted in the initial stage of the project on an old codebase that is hard to parse out and integrate into the current one. I can provide the original main script where we use LoRA if that could help. But in summary for LoRA we used the PeFT config and simply wrapped around the main model (Take a look at https://github.com/dvlab-research/LISA/blob/main/train_ds.py). 

The reason LoRA didn't do well is not clear to us either. We tried for quite some time making the LoRA model works. But we found out that the LoRA version couldn't even overfit to a few shapes. We suspect it was due to the newly added vocabulary, but we didn't do any analysis on it. 


Here is our old training script. Take a look from line 164-232 for the LoRA setup.
[train_aipparel_old.py](https://github.com/user-attachments/files/22246439/train_aipparel_old.py)

**Comment by yangmingweiwtu** [comment]
> Hi, the LoRA-finetuning experiment was conducted in the initial stage of the project on an old codebase that is hard to parse out and integrate into the current one. I can provide the original main script where we use LoRA if that could help. But in summary for LoRA we used the PeFT config and simply wrapped around the main model (Take a look at https://github.com/dvlab-research/LISA/blob/main/train_ds.py).
> 
> The reason LoRA didn't do well is not clear to us either. We tried for quite some time making the LoRA model works. But we found out that the LoRA version couldn't even overfit to a few shapes. We suspect it was due to the newly added vocabulary, but we didn't do any analysis on it.
> 
> Here is our old training script. Take a look from line 164-232 for the LoRA setup. [train_aipparel_old.py](https://github.com/user-attachments/files/22246439/train_aipparel_old.py)

Thanks for your reply!
